src/StandaloneHydratorPluginManagerFactory.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-hydrator for the canonical source repository
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-hydrator/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Zend\Hydrator;
+
+use Psr\Container\ContainerInterface;
+
+final class StandaloneHydratorPluginManagerFactory
```

Comment:
Can probably be simplified with an invokable factory?

Reply:
That would require using zend-servicemanager... and the whole point of having the standalone version is when you do not want to use that implementation.

Reply:
Doesn't the `ConfigProvider` already retrieve a naming structure that assumes `zend-servicemanager`? 

https://github.com/zendframework/zend-hydrator/pull/87/files#diff-e97918d312824c7b64da5ef255c18258

Reply:
Yes and no!

We co-opted that for containers compatible with Expressive, and document it there as well: https://docs.zendframework.com/zend-expressive/v3/features/container/config/

So if you're using any of the 5 or so other compatible implementations, this will continue to work. It will also work if you're using the component in another context entirely, which is the important bit.

Reply:
:+1: